atomisticparsers/h5md/parser.py

```diff
@@ -580,6 +580,15 @@ def parse_system(self):
             time = system_info[step].pop("time")
             atoms_dict = system_info[step]
 
+            atom_labels = atoms_dict.get("labels")
+            if atom_labels is not None:
+                try:
+                    from ase import Atoms as ase_atoms
+
+                    _ = ase_atoms(symbols=atom_labels)
```

Comment:
the specific function to test if symbols are valid in ase is 
```python
from ase.symbols import symbols2numbers
try:
    symbols2numbers(labels)
except KeyError:
    labels = ['X'] * len(labels)

Reply:
but I am also fine with trying to generate the full ase atoms

Reply:
ah, nice, thanks. done.